### ISSUE TITLE
Remove "npm test" from all templates

### ISSUE THREAD

**Issue by compulim** [microsoft/BotBuilder-Samples#3346]
tl;dr all bot templates should not include an *always-failing* `npm test` script. It should work out of the box without any changes to any code.

**Is your feature request related to a problem? Please describe.**

I am trying to deploy a bot using Azure Web App Deployment Center. Here are my steps:

1. Clone this repo
1. Create a new repo, namely `my-bot`
1. Copy `/samples/javascript_nodejs/02.echo-bot/` recursively to `my-bot` repo at root level
1. Commit `my-bot` to GitHub as `main` branch
1. Create a new Azure Web App Bot
1. Enable automatic deployment from GitHub from `my-bot` repo (in Azure Web App of the bot)
   - Behind the scene, Deployment Center just added a GitHub Actions YAML to `my-bot` repo to do the deployment
   - [Sample YML created by Azure Web App is here](https://github.com/compulim/experiment-bug-bot/blob/main/.github/workflows/main_hawo-bot-bug-3967.yml)

Then, it fail because the `02.echo-bot` is trying to run `npm run test --if-present`, which failed.

```
> echo "Error: no test specified" && exit 1

"Error: no test specified" 
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! echobot@1.0.0 test: `echo "Error: no test specified" && exit 1`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the echobot@1.0.0 test script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

**Describe the solution you'd like**

If `02.echo-bot` doesn't come any test harness (e.g. Mocha/Jest), it should not include an always-failing "npm test" script.

**Describe alternatives you've considered**

- Add a simple Mocha test
- Remove it myself every time I clone the repo

**Additional context**

It is very reasonable for a CI/CD pipeline to run `npm test` to make sure the deployed bit looks okay. I think Azure Web App Deployment Center is doing it correctly by running `npm test` at deployment time.

**Comment by compulim** [comment]
Ping.
